editor.html

```diff
@@ -345,7 +345,7 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
                                 <li><span title="Spanish"><a lang="es" href="#" class="lang-choice" id="es" tabindex="122">Español</a></span></li>
                                 <li><span title="Polish"><a lang="pl" href="#" class="lang-choice" id="pl" tabindex="123">Polski</a></span></li>
                             </ul>
-                            <div><a id="add-language-link" class="action" title="Add a language translation to the Editor" href="https://github.com/bbcmicrobit/PythonEditor/blob/v2-beta/docs/translations.md#12-translating-the-editor" target="_blank" tabindex="124">Add a language</a></div>
+                            <div><a id="add-language-link" class="action" title="Add a language translation to the Editor" href="https://support.microbit.org/a/solutions/articles/19000106022-translating-the-micro-bit-python-editor" target="_blank" tabindex="124">Add a language</a></div>
```

Comment:
Whenever I use a link with a `...org/a/solutions/...` without being logged in, I get a flash of the freshdesk dashboard layout (no content, but can clearly see the page background colours) and then get immediately redirected to URL without the `/a/` which is loaded in the normal Knowledge-base style.
Should we use URLs without the `/a/` instead? Or is there a good reason to use this style URL?